packages/utils/src/browser.ts

```diff
@@ -108,7 +108,7 @@ function _htmlElementAsString(el: unknown, keyAttrs?: string[]): string {
       }
     }
   }
-  const allowedAttrs = ['type', 'name', 'title', 'alt'];
+  const allowedAttrs = ['aria-label', 'aria-labelledby', 'type', 'name', 'title', 'alt'];
```

Comment:
I think `aria-labelledby` refers to a different dom element, so would not be too helpful here? as that would be the ID of the element that has the label.